etc/org.opencastproject.organization-mh_default_org.cfg

```diff
@@ -359,6 +359,11 @@ prop.admin.shortcut.add_media.previous_tab=shift+alt+left
 # Default: /editor-ui/index.html?id=$id
 #prop.admin.editor.url=/editor-ui/index.html?id=$id
 
+# Enable themes in the admin intterface.
+# Format: boolean
+# Default: false
+#prop.admin.themes.enabled=false
```

Comment:
The new default is `true` so we do not change anything in `r/15.x` but we want this to be disabled by default in 16. So we do that via this configuration:

```suggestion
# Default: true
prop.admin.themes.enabled=false
```